contrib/storage-hive/core/src/test/java/org/apache/drill/exec/impersonation/hive/TestStorageBasedHiveAuthorization.java

```diff
@@ -145,6 +148,41 @@ private static void setStorabaseBasedAuthorizationInHiveConf() {
     return hiveConfig;
   }
 
+  /**
+   * User       Groups
+   * <br/>
```

Comment:
I will put a new comment, since the previous one is swallowed by your commit force-pushing.

You didn't get an idea, I suggested to add `</br>` for every line of your table (use `ctrl+q` for checking it in Intelij).
But forgot about that, the proper way is to use `<table>`, `<tr>`, `<td>` tags. Please follow it.

Reply:
changed to multi-line comment

Reply:
looks good. There is no need in tags and annotations anymore

Reply:
fixed